### PR TITLE
Changed continuous variables measures to be patient based.

### PR DESCRIPTION
**Pull request by RohitKandimalla**
MAT-2709
## Description

- The continuous variable measures are changed to be patient-based.
- When creating a new measure, the patient-based dropdown used to have only "NO" for the continuous variable measure.
- Now the changes have been made for CV measures to be patient-based.
- The same works for the patient-based input field on the Measure General information details page.
## JIRA Ticket

https://jira.cms.gov/browse/MAT-2709
## Checklist

- [ ] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [ ] Tests are included and test edge cases.
- [ ] Tests have been run locally and pass.

## Screenshots (if appropriate)
- [ ] None applicable

### CODE REVIEW

File: src/main/java/mat/client/shared/MatContext.java [hunk 1]
```diff
@@ -1530,8 +1531,12 @@ public void setPopulationBasisList(ListBox listBox, List<String> list, String de
     public List<String> getPatientBasedIndicatorOptions(String measureScoringMethod) {
         List<String> patientBasedList = new ArrayList<>();
         patientBasedList.add("No");
-        if (!MatConstants.CONTINUOUS_VARIABLE.equalsIgnoreCase(measureScoringMethod)) {
+        if (!ModelTypeHelper.isFhir(getCurrentMeasureModel())) {
             patientBasedList.add("Yes");
+        } else {
+            if (!MatConstants.CONTINUOUS_VARIABLE.equalsIgnoreCase(measureScoringMethod)) {
+                patientBasedList.add("Yes");
+            }
```
Comment (jkotanchik-SB) [inline src/main/java/mat/client/shared/MatContext.java:1539]: Suggest combining into single `else if` instead of nesting.

File: src/main/java/mat/client/shared/MatContext.java [hunk 2]
```diff
@@ -1530,8 +1531,12 @@ public void setPopulationBasisList(ListBox listBox, List<String> list, String de
...
         }
```
Comment (jkotanchik-SB) [inline src/main/java/mat/client/shared/MatContext.java:1540]: Suggest combining into single `else if` instead of nesting. Simply removes a linebreak and a couple of square brackets when reading through the code.
```suggestion
        } else if (!MatConstants.CONTINUOUS_VARIABLE.equalsIgnoreCase(measureScoringMethod)) {
                patientBasedList.add("Yes");
        }
```

**Review by jkotanchik-SB — APPROVED** [review]
